SUPPORT.md

```diff
@@ -1,57 +1,26 @@
 OpenSSL User Support resources
 ==============================
 
-_Under Construction; not more than a collection of text fragments yet._
-
 See the OpenSSL website www.openssl.org for details on how to obtain
-commercial technical support. Free community support is available through the
-openssl-users email list (see
-<https://www.openssl.org/community/mailinglists.html for> further details).
-
-If you have any problems with OpenSSL then please take the following steps
-first:
-
- - Download the latest version from the repository
-   to see if the problem has already been addressed
- - Configure with no-asm
- - Remove compiler optimization flags
-
-If you wish to report a bug then please include the following information
-and create an issue on GitHub:
-
- - OpenSSL version: output of 'openssl version -a'
- - Configuration data: output of 'perl configdata.pm ==dump'
- - OS Name, Version, Hardware platform
- - Compiler Details (name, version)
- - Application Details (name, version)
- - Problem Description (steps that will reproduce the problem, if known)
- - Stack Traceback (if the application dumps core)
+commercial technical support.
 
-Just because something doesn't work the way you expect does not mean it
-is necessarily a bug in OpenSSL. Use the openssl-users email list for this type
-of query.
-
-For *questions* on how to use OpenSSL or what went wrong when you
-tried something, our primary resource is the mailing list
-openssl-users@openssl.org, where you can get help from others in the
-OpenSSL community (which includes the developers as time permits).
-
-Only subscribers can post to openssl-users@openssl.org (although the
-archives are public).
-For more information, see <https://www.openssl.org/community/mailinglists.html>
-
-You have general questions about using OpenSSL
-----------------------------------------------
+If you have general questions about using OpenSSL
+-------------------------------------------------
 
 In this case the [openssl-users] mailing list is the right place for you.
 The list is not only watched by the OpenSSL team members, but also by many
 other OpenSSL users. Here you will most likely get the answer to your questions.
 An overview over the [mailing lists](#mailing-lists) can be found below.
 
-You found a Bug
----------------
+If you think you found a Bug
+----------------------------
 
-If you have any problems with OpenSSL then please take the following steps first:
+*NOTE: this section assumes that you want to report it or figure it out and
+fix it.  What's written here is not to be taken as a recipe for how to get a
+working production installation*
```

Comment:
I still don't particularly like this. If they search the mailing lists, and download the latest version and their problem is fixed - then great - go ahead use it!

My objection would be better handled by moving the no-asm suggestion out of this section or removing it altogether. I'd also be tempted to remove the compiler optimization flags thing (how many problems do we see that would be solved by this? Some, but not many).

Note: I'm not objecting to this PR. I'd approve it with this text in it (overall the PR is an improvement on what is there). I just think it could be even better.

Reply:
> (how many problems do we see that would be solved by this? Some, but not many)

Historically, we've had a fair number of interactions that repeat "what happens if you configure with `no-asm`?" as well as "what happens if you remove these optimization flags?".  It was apparently enough to make it into a general text.

> Note: I'm not objecting to this PR. I'd approve it with this text in it (overall the PR is an improvement on what is there). I just think it could be even better.

There's always space for improvement, of course.  And someone will read this again in some time and propose new changes (perhaps you :wink:), after having mulled over it for a bit.

Reply:
So, can I take that as this concern resolved for now?

Reply:
Resolved as far as this PR is concerned. The issue I'm complaining about remain IMO, but it doesn't need to be within the scope of this PR if you don't want to address it.

Reply:
That's all I'm asking.  Thanks